app/web/src/components/ApplyChangeSetButton.vue

```diff
@@ -1,29 +1,6 @@
 <template>
   <VButton
-    v-if="
-      featureFlagsStore.DEV_SLICE_REBASING && statusWithBase?.conflictsWithBase
```

Comment:
I think we can't just delete this... we need the conflicts check

Reply:
gotcha! So the conflicts isn't just limited to the DEV_SLICE_REBASING! Thanks - will fix now

Reply:
@jobelenus fixed up now! I just deleted the rebase button and the checks for the flag